### PR TITLE
example screens

### PR DESCRIPTION
**Pull request by tomstumshais**
# Description

Add `example-screens` folder in `mock-api/resources` folder. It contains those screen examples:
- `address`
- `dob`
- `email`
- `name&gender`
- `optins`
- `payment`
- `payment-invoice`
- `phone`
- `policyholder`
- `success`
- `summary`

Each screen folder contains `manifest.json`, `field_definition.json` with common validation rules for fields and `contents.csv` which contains everything from `manifest.json` + `field_definition` rules. Some of those screens is not needed `field_definitions`.

Please review everyone if you are happy with provided format. Feel free to suggest any ideas. ✌️ 

**To remember: `example-screens` folder should be deleted when webapp goes live.**

Screenshot 2020-07-23 at 13 43 27

# Checklist
- [ ] I have tested this code locally
- [ ] I have written unit tests
- [x] I looked over the submitted code and I am happy with the quality of it
- [ ] I have updated [the product documentation](https://github.com/kasko/docs)

_Please always fill in all the checkmarks. If some of them are not relevant, then please strikethrough them (~ symbol) and write a comment why it's not relevant._

### CODE REVIEW

File: mock-api/resources/example-screens/optins/manifest.json [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+    },
+    {
+      "type": "form",
+      "config": {
+        "width": "wide",
+        "components": [
+          {
+            "type": "form-input-optin",
+            "config": {
+              "field_name": "optin1",
```
[the remark this answers is not in the document]
Comment (tomstumshais) [reply mock-api/resources/example-screens/optins/manifest.json:18]: Good point! 👍

File: mock-api/resources/example-screens/email/field_definition.json [hunk 2]
```diff
@@ -0,0 +1,7 @@
+[
+  {
+    "path": null,
+    "name": "email",
+    "validation": "required|string|email|max|regex"
```
Comment (mkasparsone) [inline mock-api/resources/example-screens/email/field_definition.json:5]: should we just leave `regex` and `max` without an example??? (like, max:100, or regex:/^[0-9]$/

Comment (tomstumshais) [reply mock-api/resources/example-screens/email/field_definition.json:5]: Good point! 👁️

**Comment by ozolsg** [comment]
Hmm, maybe we should update the `example-screens` with something like `example-screens-delete` but with better naming  to minimise the steps for f/e team to remember? Or put a checkmark before submitting PR to delete the folder?

**Comment by tomstumshais** [comment]
> Hmm, maybe we should update the `example-screens` with something like `example-screens-delete` but with better naming to minimise the steps for f/e team to remember? Or put a checkmark before submitting PR to delete the folder?

Good point 👍  I think it's better just to add `delete` to folders naming than add one more checkbox in PR list, because this list is used for each change request, but this folder should be gone after webapp's initial version.

**Comment by tomstumshais** [comment]
Renamed folder to `example-screens_DELETE-ME`.
Feel free to give any other ideas for naming if it sounds wrong. 🤞

**Pull request merged**